common/windows/delphi/ext/cef4delphi/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "${THIS_SCRIPT%/*}/../../../../../resources/build/builder.inc.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+builder_describe "cef4delphi component" clean configure build test
+builder_parse "$@"
+
+#-------------------------------------------------------------------------------------------------------------------
+
+source "$KEYMAN_ROOT/resources/build/win/environment.inc.sh"
+
+builder_describe_outputs \
+  configure:project    /resources/build/win/delphi_environment_generated.inc.sh \
+  build:project        /common/windows/lib/CEF4Delphi.bpl
+
+#-------------------------------------------------------------------------------------------------------------------
+
+function do_build() {
+  "$DEVTOOLS" -ai "$THIS_SCRIPT_PATH/source"
+  delphi_msbuild packages/CEF4Delphi.dproj "//p:Platform=Win32"
+  "$DEVTOOLS" -ip "$COMMON_OUTLIB/CEF4Delphi.bpl"
+}
+
+builder_run_action clean:project        clean_windows_project_files
```

Comment:
It could be because the project has been updated to have outputs in `obj` and `bin` which will be cleaned up by `configure_windows_build_environment`

Reply:
Ok, the `.dproj` changes make more sense to me now.  Still...

```
        <DCC_DcpOutput>..\..\..\..\lib</DCC_DcpOutput>
        <DCC_BplOutput>..\..\..\..\lib</DCC_BplOutput>
```

Are the paths in that file explicitly to locations within a top-level `bin` and `obj`?  If so, that's not clear to me, but that would make sense.  (I don't see anything implying that either of those two values are based on `<DCC_DcuOutput>` or `<DCC_ExeOutput>`... and the `../../../../` relative pathing seen above doesn't help that concern.)

Reply:
Yeah this was mostly around cleanup as @rc-swag noted.

The ../../../../lib path is relative to the .dproj file, thus the .dcp and .bpl files are now written to /common/lib (which removes a silly copy step), and are not removed on a clean action, because the Delphi IDE depends on them and throws stupid errors when they go missing.